api-definitions/db_positions/api.yaml

```diff
@@ -200,6 +266,61 @@ parameters:
       See [pagoPa specs](https://www.agid.gov.it/sites/default/files/repository_files/specifiche_attuative_pagamenti_1_3_1_0.pdf) for more info on this field and the IUV.
     type: string
     pattern: "^[0123][0-9]{17}$"
+    maxLength: 50
+  AmountFrom:
```

Comment:
i'm not sure. Is there a way to define a custom type definition and use it for differents query params ?
in a nutshell `AmountFrom` and `AmountTo ` change only name but are same type.

Reply:
It is possible to link body params to the "Amount" definition, but we did not find a way to link query parameters to it. Also,  "AmountFrom" and "AmountTo" still need different descriptions.